src/fortune-wheel-booth-frontend/src/costants.ts

```diff
@@ -47,12 +47,12 @@ export const PRIZES: WheelDataType[] = [
     style: { backgroundColor: '#564B9E' },
   },
   {
-    option: 'merch',
+    option: 'merch5',
     image: { uri: Astronaut, sizeMultiplier: 0.7, offsetY: 220 },
     style: { backgroundColor: '#FBB03C' },
   },
   {
-    option: 'merch',
+    option: 'merch6',
```

Comment:
By changing the names in this way, we never hit the `PRIZES_VALUES_MAPPING.merch` value anymore

Reply:
You mixed up between `PRIZES_VALUES_MAPPING` and `PRIZES`